models/chunter/src/index.ts

```diff
@@ -129,6 +142,46 @@ export function createModel (builder: Builder): void {
     direct: false
   })
 
+  registerMapper(chunter.mapper.CommentNotificationMappes, {
```

Comment:
It should be called from server, since model code is executed once outside of server process to create model.